framework/src/modules/chain/chain.js

```diff
@@ -237,7 +237,15 @@ module.exports = class Chain {
 					action.params[0],
 					action.params[1]
 				),
-			getSlotsHelper: async () => this.slots,
+			getSlotTime: async action =>
+				action.params
+					? this.slots.getTime(action.params[0])
+					: this.slots.getTime(),
+			getSlotNumber: async action =>
+				action.params
+					? this.slots.getSlotNumber(action.params[0])
```

Comment:
Would you like to use named parameters? This has just been merged into dev. Ref: https://github.com/LiskHQ/lisk/pull/3065

`getSlotNumber(action.params.num)`